### PR TITLE
refactor(bridge): Make use of new Git API model

### LINKED ISSUE #8046: Bridge :: Make use of new Git API model
## Technical Details

In #7963, the payload has changed to split the data between HTTPS and SSH. The `bridge-client` should be updated to make use of this new format. 

## DoD

- [x] The new API model can be ingested
- [x] Test are adapted
- [x] Mocks are adapted

## Dependencies

- #7963

### PR DESCRIPTION
**Pull request by Kirdock**
Closes #8046 
Signed-off-by: Klaus Strießnig <k.striessnig@gmail.com>

### CODE REVIEW

File: bridge/client/app/_components/ktb-project-settings/ktb-project-settings.component.ts [hunk 1]
```diff
@@ -212,82 +194,43 @@ export class KtbProjectSettingsComponent implements OnInit, OnDestroy, PendingCh
...
-            );
-            this.isCreatingProjectInProgress = false;
-          }
+    const shipyardBase64 = btoa(fileContent);
+    const projectName = this.projectNameControl.value;
+
+    this.dataService.createProjectExtended(projectName, shipyardBase64, this.gitDataExtended ?? undefined).subscribe(
+      () => {
+        this.projectName = projectName;
+        this.dataService.loadProjects().subscribe(() => {
```
Comment (heinzburgstaller) [inline bridge/client/app/_components/ktb-project-settings/ktb-project-settings.component.ts:213]: Can we refactor the the nested subscription to to a RXJS pipe and then subscribe once to it?

Comment (Kirdock) [reply bridge/client/app/_components/ktb-project-settings/ktb-project-settings.component.ts]: done

File: bridge/client/app/_views/ktb-error-view/ktb-error-view.component.html [hunk 2]
```diff
@@ -12,6 +12,10 @@ <h2>Permission denied</h2>
         <h2>Action not permitted</h2>
         <p>User is not allowed to perform this action.</p>
       </ng-container>
+      <ng-container *ngSwitchCase="ServerErrors.CONFIGURATION_SERVICE_NOT_ALLOWED">
+        <h2>Action not permitted</h2>
+        <p>You are not allowed to perform this action if the configuration service is enabled.</p>
```
Comment (renepanzar) [inline bridge/client/app/_views/ktb-error-view/ktb-error-view.component.html:17]: When would that happen? Is the error message correct? I would imply I get this error when the configuration service is **not enabled**?

**Review by renepanzar — COMMENTED** [review]
Just a question about an error message, the rest looks good to me.

File: bridge/client/app/_components/ktb-project-settings/ktb-project-settings-git-extended/ktb-project-settings-git-extended.component.html [hunk 3]
```diff
@@ -1,20 +1,14 @@
...
-      class="mr-2"
-      uitestid="ktb-no-upstream-form-button"
-    >
-      No upstream
-    </dt-radio-button>
-  </div>
-  <ng-template #noUpstreamOverlay
-    >This project has already an upstream configured. If an upstream is already configured, it can not be unset.
-  </ng-template>
+  <dt-radio-button
```
[the remark this answers is not in the document]
Comment (renepanzar) [reply bridge/client/app/_components/ktb-project-settings/ktb-project-settings-git-extended/ktb-project-settings-git-extended.component.html:3]: We had a conversation IRL, the upstream being not configured state is not relevant any longer in future Keptn versions.

**Review by renepanzar — APPROVED** [review]
LGTM!

**Pull request merged**